docs/pages/guides/mesheryctl/authenticate-with-meshery-via-cli.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Authenticating with Meshery via CLI 
+permalink: guides/mesheryctl/authenticate-with-meshery-via-cli
+language: en
+type: Guides
+category: mesheryctl
+---
+
+To authenticate with Meshery via mesh you have to use the command `mesheryctl system login`. On execution of this command you get a an option to choose a provider, you would have to choose `Meshery` as your provider to authenticate yourself. After doing so, a new window opens up with ways to login. 
```

Comment:
```suggestion
To authenticate with Meshery through `mesheryctl` you will use the command `mesheryctl system login`. Upon execution of this command, select your Provider of choice, then authenticate to your chosen Provider.
```

Reply:
It'd be good to include a tip / note "What is a Meshery Provider?" using `{% include alert.html %}`